Meadowlark-js/packages/meadowlark-core/test/extraction/DocumentReferenceExtractor.test.ts

```diff
@@ -648,3 +648,105 @@ describe('when extracting with school year reference in body', () => {
     `);
   });
 });
+
+describe('when extracting with a school year in a reference document', () => {
```

Comment:
```suggestion
describe('when extracting with school year in a reference document', () => {
```

I think it's better to have a similar name to the other test for easier location